### PR TITLE
Plenty 317

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: resources/js/src/app/components/itemList/filter/ItemColorTiles.js [hunk 1]
```diff
@@ -19,6 +19,14 @@ Vue.component("item-color-tiles", {
         })
     },
 
+    mounted() {
+        this.$nextTick(function () {
+            $(function() {
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemColorTiles.js:24]: Do we actually need the `$(function() {...` wrapper, now that we use `mounted`?

File: resources/views/ItemList/Components/Filter/ItemFilterImage.twig [hunk 2]
```diff
@@ -4,7 +4,7 @@
             <div class="form-check" v-for="value in facet.values" :key="value.id">
                 <input :id="'option-' + value.id" class="form-check-input hidden-xs-up" type="checkbox" :checked="value.selected" @change="updateFacet(value)" :disabled="isLoading">
                 <label :for="'option-' + value.id" class="form-check-label" rel="nofollow">
-                    <img :src="value.imageUrl" @error="handleImageError($event)" width="80px"/>
+                    <img :src="value.imageUrl" @error="handleImageError($event)" :width="value.imageUrl ? '80px' : ''"/>
```
Comment (TheKeymaster) [inline resources/views/ItemList/Components/Filter/ItemFilterImage.twig:7]: How would this look like? Can you add a screenshot of before and after, so we can decide which one we want to use.

File: resources/views/content/scripts.twig [hunk 3]
```diff
@@ -32,12 +32,6 @@
     <script src="{{ plugin_path("Findologic") }}/js/dist/filters-component-min.js"></script>
 {% endif %}
 
-<script>
```
Comment (TheKeymaster) [inline resources/views/content/scripts.twig:35]: :+1:

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/views/ItemList/Components/Filter/ItemFilterImage.twig (hunk 2 above)
Comment (RimantasAdeoWeb) [reply resources/views/ItemList/Components/Filter/ItemFilterImage.twig:7]: Before empty space would be left after the filter is selected for the image, even when there is no image:
https://snipboard.io/8YVMkm.jpg
After:
https://snipboard.io/3FE1IH.jpg

File: resources/js/src/app/components/itemList/filter/ItemColorTiles.js (hunk 1 above)
Comment (RimantasAdeoWeb) [reply resources/js/src/app/components/itemList/filter/ItemColorTiles.js]: Apparently not, works well without it.

**Review by TheKeymaster — APPROVED** [review]

File: resources/views/ItemList/Components/Filter/ItemFilterImage.twig (hunk 2 above)
Comment (TheKeymaster) [reply resources/views/ItemList/Components/Filter/ItemFilterImage.twig:7]: Thank you, we will leave it as it is.

**Pull request merged**